### ISSUE TITLE
keras.utils.to_categorical(y_train, num_classes)  function  trouble

### ISSUE THREAD

**Issue by struggling-man** [keras-team/keras#10254]
y_train = keras.utils.to_categorical(y_train, num_classes)
y_test = keras.utils.to_categorical(y_test, num_classes)

I don't know why to_categorical performs multiply operation，for example the result in  mnist dataset 

550000 100000
10000 test samples 10000
100000 test label 100000
I couldn't find the problem. I need you help! thank you!

**Comment by struggling-man** [comment]
I can't understand that the model can be trained when I don't use to_categorical operation.
